packages/gx-agent/src/agent/GXComplianceClient.ts

```diff
@@ -111,13 +113,14 @@ export class GXComplianceClient implements IAgentPlugin {
         verifiableCredentials: [selfDescribedVC],
         challenge: GXComplianceClient.getDateChallenge(),
         domain: signInfo.participantDomain,
-        persist: true,
+        persist: args.persist ? args.persist : false,
```

Comment:
Not sure why this is needed everywhere, simply pass in args.persist

Reply:
changed to just: `args.persist`